archetypes/archetype-tools/src/test/resources/serviceMapping.vm

```diff
@@ -0,0 +1 @@
ventbridge": "EventBridge", "machinelearning": "MachineLearning", "s3control": "S3Control", "transfer": "Transfer", "resourcegroups": "ResourceGroups", "support": "Support", "fms": "Fms", "ioteventsdata": "IotEventsData", "swf": "Swf", "kinesisvideosignaling": "KinesisVideoSignaling", "cloudsearchdomain": "CloudSearchDomain", "workmailmessageflow": "WorkMailMessageFlow", "acmpca": "AcmPca", "servicediscovery": "ServiceDiscovery", "rdsdata": "RdsData", "translate": "Translate", "athena": "Athena", "applicationinsights": "ApplicationInsights", "ram": "Ram", "lexmodelbuilding": "LexModelBuilding", "ec2instanceconnect": "Ec2InstanceConnect", "codeguruprofiler": "CodeGuruProfiler", "mturk": "MTurk", "cloudfront": "CloudFront", "ec2": "Ec2", "sagemakera2iruntime": "SageMakerA2IRuntime", "route53resolver": "Route53Resolver", "managedblockchain": "ManagedBlockchain", "kinesisanalytics": "KinesisAnalytics", "kendra": "Kendra", "identitystore": "Identitystore", "codepipeline": "CodePipeline", "elastictranscoder": "ElasticTranscoder", "organizations": "Organizations", "pi": "Pi", "servicecatalog": "ServiceCatalog", "apigatewayv2": "ApiGatewayV2", "servicequotas": "ServiceQuotas", "marketplacemetering": "MarketplaceMetering", "batch": "Batch", "codecommit": "CodeCommit", "mq": "Mq", "iotsecuretunneling": "IoTSecureTunneling", "costandusagereport": "CostAndUsageReport", "signer": "Signer", "migrationhubconfig": "MigrationHubConfig", "codestarnotifications": "CodestarNotifications", "elasticloadbalancing": "ElasticLoadBalancing", "pricing": "Pricing", "kinesisvideomedia": "KinesisVideoMedia", "rekognition": "Rekognition", "cloudtrail": "CloudTrail", "elasticloadbalancingv2": "ElasticLoadBalancingV2", "personalizeevents": "PersonalizeEvents", "ivs": "Ivs", "mediatailor": "MediaTailor", "cloudhsm": "CloudHsm", "honeycode": "Honeycode", "cloudsearch": "CloudSearch", "acm": "Acm", "timestreamwrite": "TimestreamWrite", "inspector": "Inspector", "lakeformation": "LakeFormation", "elasticinference": "ElasticInference", "schemas": "Schemas", "amplify": "Amplify", "globalaccelerator": "GlobalAccelerator", "s3": "S3", "clouddirectory": "CloudDirectory", "accessanalyzer": "AccessAnalyzer", "greengrass": "Greengrass", "worklink": "WorkLink", "sagemaker": "SageMaker", "config": "Config", "codeartifact": "Codeartifact", "workspaces": "WorkSpaces", "savingsplans": "Savingsplans", "sqs": "Sqs", "iot": "Iot", "codestarconnections": "CodeStarConnections", "ssoadmin": "SsoAdmin", "sfn": "Sfn", "directory": "Directory", "marketplacecommerceanalytics": "MarketplaceCommerceAnalytics", "sns": "Sns", "mediastoredata": "MediaStoreData", "workmail": "WorkMail", "cognitoidentityprovider": "CognitoIdentityProvider", "codedeploy": "CodeDeploy", "iotjobsdataplane": "IotJobsDataPlane", "mobile": "Mobile", "docdb": "DocDb", "neptune": "Neptune", "wafv2": "Wafv2", "glacier": "Glacier", "apigatewaymanagementapi": "ApiGatewayManagementApi", "fsx": "FSx", "sagemakerruntime": "SageMakerRuntime"})
```

Comment:
How is this file updated with new services?

Reply:
This is just used for testing. serviceMapping.vm is created at build time